build.zig.ini

```diff
@@ -0,0 +1,8 @@
+[package]
+name=jui
+version=0.1
+
+[dependency]
+name=cf
+url=https://github.com/desttinghim/cf/archive/d5dfac3e00dfaa2e36c86c006f8fd33c3131773d.tar.gz
```

Comment:
Could you backport the cf changes to `zig-java/cf` and pull from there? :)

Reply:
Yes, though I should probably remove the `build.zig.ini` because the package manager mvp didn't include a way to depend on zig code.

Reply:
Probably a good idea for now, yeah. Submodules will do the trick.